osu.Server.Spectator/osu.Server.Spectator.csproj

```diff
@@ -14,12 +14,12 @@
         <PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" Version="6.0.10" />
         <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.10" />
         <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />
-        <PackageReference Include="ppy.osu.Game" Version="2022.1117.0" />
-        <PackageReference Include="ppy.osu.Game.Rulesets.Catch" Version="2022.1117.0" />
-        <PackageReference Include="ppy.osu.Game.Rulesets.Mania" Version="2022.1117.0" />
-        <PackageReference Include="ppy.osu.Game.Rulesets.Osu" Version="2022.1117.0" />
-        <PackageReference Include="ppy.osu.Game.Rulesets.Taiko" Version="2022.1117.0" />
-        <PackageReference Include="ppy.osu.Server.OsuQueueProcessor" Version="2022.812.0" />
+        <PackageReference Include="ppy.osu.Game" Version="2022.1205.0" />
+        <PackageReference Include="ppy.osu.Game.Rulesets.Catch" Version="2022.1129.0" />
+        <PackageReference Include="ppy.osu.Game.Rulesets.Mania" Version="2022.1129.0" />
+        <PackageReference Include="ppy.osu.Game.Rulesets.Osu" Version="2022.1129.0" />
+        <PackageReference Include="ppy.osu.Game.Rulesets.Taiko" Version="2022.1129.0" />
```

Comment:
Probably nuget being slow. I'll update them, although it shouldn't matter too much for the purpose of this bump.

Reply:
Not sure why but [50cd5e8](https://github.com/ppy/osu-server-spectator/pull/152/commits/50cd5e83d19384eb3323b804756f8a7bba143b6a) only bumped three out of four. I did the last one in [a84c92f](https://github.com/ppy/osu-server-spectator/pull/152/commits/a84c92f20ae6121b60bf7b89af81f2045379d748) because I didn't see a reason why that would be the case.